libs/api/domains/driving-license/src/lib/drivingLicense.service.ts

```diff
@@ -355,8 +355,15 @@ export class DrivingLicenseService {
     }
   }
 
-  async canApplyFor(type: 'B-full' | 'B-temp' | 'BE', token: string) {
-    if (type === 'B-full') {
+  async canApplyFor(
+    type: 'B-full' | 'B-temp' | 'BE' | 'B-full-renewal-65',
```

Comment:
Should this be an enum/union type at this point?

Reply:
yes, might be a good idea to refactor in another PR